amundsen_application/static/js/components/AvatarLabelList/styles.scss

```diff
@@ -0,0 +1,152 @@
+@import 'variables';
+
+.editable-list-component {
```

Comment:
Can we change class names to reflect the new component name?

Reply:
Oversight on my part